client/src/pages/guide/english/nodejs/file-system/index.md

```diff
@@ -55,12 +55,12 @@ Node.js code to write content into file.
 ```javascript
 const fs = require('fs');
 fs.writeFile('output.txt', "New content added", (err, data) => {
-	if(err){
-		console.log(err);
-	}
-	else{
-		console.log("The file is saved");
-	}
+    if (err) {
+        console.log(err);
+    }
+    else {
+        console.log("The file is saved");
+    }
 });
 ```
```

Comment:
Hi @FarhanYaseen 

Please move the proposed changes in #19212 over to this PR.